server/src/repositories/patient_repository.ts

```diff
@@ -51,4 +52,27 @@ export class PatientRepository {
         const res = await client.query(sql, [patientId]).finally(() => client.release());
         return res.rows[0]?.assigned_doctor_id;
     }
+
+    async findPatientCounts(): Promise<PatientCount[]> {
+        const client = await this.pool.connect();
+
+        const sql = `
+            SELECT user_id, first_name, last_name, email, count(*)
```

Comment:
if you could use aliases that would make it easier to read in the future.

Reply:
You right, will update in another pr. I usually do a bad job of this I need to start using shorter alias too.